py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/convert_gtfs.py

```diff
@@ -0,0 +1,53 @@
+import zipfile
```

Comment:
should be a pretty straight forward file.

get the `ZipFile` either locally or from s3. parse out the start date timestamp, generate a table from each text file in the zip, adding in start time columns to each one.